nats/subsz.yml

```diff
@@ -0,0 +1,60 @@
+type: json
+data:
+  url: /subsz
+  server: ${NATS_HOSTPORT}
+
+observation:
+  - name: subsz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: subscriptions
+        source: num_subscriptions
+        type: long_gauge
+        label: subscriptions
+        description: The number of subscriptions
+
+      - name: inserts
+        source: num_inserts
+        type: long_gauge
+        label: inserts
+        description: The number of insert operations in subscriptions list
+
+      - name: removes
+        source: num_removes
+        type: long_gauge
+        label: removes
+        description: The number of remove operations in subscriptions list
+
+      - name: matches
+        source: num_matches
+        type: long_gauge
```

Comment:
Are you sure this is a gauge? It sounds like some kind of a cumulative.

Reply:
I will check the real value again, not sure i check it's value last time.

Reply:
It is counter, i also change some other metric type to counter